mempool/mempool.go

```diff
@@ -1134,14 +1134,39 @@ func (mp *TxPool) maybeAcceptTransaction(tx *dcrutil.Tx, isNew, rateLimit, allow
 	// Votes that are on too old of blocks are rejected.
 	if isVote {
 		_, voteHeight := stake.SSGenBlockVotedOn(msgTx)
-		if (int64(voteHeight) < nextBlockHeight-maximumVoteAgeDelta) &&
-			!mp.cfg.Policy.AllowOldVotes {
-
-			str := fmt.Sprintf("transaction %v votes on old "+
-				"block height of %v which is before the "+
-				"current cutoff height of %v",
-				tx.Hash(), voteHeight, nextBlockHeight-maximumVoteAgeDelta)
-			return nil, txRuleError(wire.RejectNonstandard, ErrOldVote, str)
+
+		switch mp.cfg.ChainParams.Name {
```

Comment:
Hmm, I really think this should be a configuration option specified when creating the `mempool`.  Checking for specific networks is quite brittle and I've been trying to avoid it in all new code.